src/SFA.DAS.CaptureTheFlag.Web/Views/Home/Index.cshtml

```diff
@@ -1,14 +1,24 @@
-﻿@{
+﻿@using Microsoft.AspNetCore.Identity
+@inject SignInManager<IdentityUser> SignInManager
+@{
     ViewData["Title"] = "Home Page";
 }
-   
+
 <div class="govuk-main-wrapper">
-    <div class="govuk-girid-row">
+    <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-xl">Welcome to Capture The Flag</h1>
 
-
-            <a role="button" class="govuk-button govuk-button--start" asp-controller="SignIn" asp-action="Index">Play now </a>
+            @if (SignInManager.IsSignedIn(User))
+            {
+                <form action="/Game/FindGame" method="get">
+                    <button class="govuk-button" type="submit">Find a new game</button>
+                </form>
+            }
+            else
+            {
+                <p>Login or Register to start a new game.</p>
```

Comment:
Instead of putting the Log in/Create account in the header where it is now, I would make "Login" and "Resister" on this line links to those pages otherwise this line really service no purpose.